src/main/groovy/org/ajoberstar/grgit/auth/AuthConfig.groovy

```diff
@@ -97,11 +99,13 @@ class AuthConfig {
     static final String FORCE_OPTION = 'org.ajoberstar.grgit.auth.force'
     static final String USERNAME_OPTION = 'org.ajoberstar.grgit.auth.username'
     static final String PASSWORD_OPTION = 'org.ajoberstar.grgit.auth.password'
+	static final String PASSPHRASE_OPTION = 'org.ajoberstar.grgit.auth.passphrase'
```

Comment:
Having these show up as parallel options (referring to the system prop naming) seems a little confusing. I think we should keep the username/password use case separate from the private key and passphrase option.